templates/editDepositPointForm.tpl

```diff
@@ -30,25 +34,29 @@
 		{fbvFormSection for="name" title="plugins.generic.sword.depositPoints.name"}
 			{fbvElement type="text" id="name" name="name" value=$name multilingual=true}
 		{/fbvFormSection}
-		
+
 		{fbvFormSection for="swordUrl" title="plugins.importexport.sword.depositUrl"}
 			{fbvElement type="text" id="swordUrl" value=$swordUrl}
 		{/fbvFormSection}
-		
+
 		{fbvFormSection for="swordUsername" title="user.username"}
-			{fbvElement type="text" id="swordUsername" value=$swordUsername}
+			{fbvElement type="text" id="swordUsername" value=$swordUsername data-error="{translate key='plugins.generic.sword.formErrors.username'}"}
 		{/fbvFormSection}
-		
+
 		{fbvFormSection for="swordPassword" title="user.password"}
-			{fbvElement type="text" password="true" id="swordPassword" value=$swordPassword}
+			{fbvElement type="text" password="true" id="swordPassword" value=$swordPassword data-error="{translate key='plugins.generic.sword.formErrors.password'}"}
```

Comment:
I'm not sure `data-error` gets passed along to the generated markup here -- are you sure this is operative?

Reply:
It does seem to work:
![image](https://user-images.githubusercontent.com/1626547/58448190-16c66180-80d5-11e9-9476-9847cef13bbc.png)

Reply:
Gotcha, yes, it does appear to be passed along. It's not a pattern we're using elsewhere (we use server-side validation, using the `FormValidator...` PHP classes, plus some front-end validation courtesy of the JQuery `.validate()` function, but these are very old structures and I'm not averse to new patterns. @NateWr, any opinion?

Reply:
Yeah I have not seen that used before either. :joy: Is it an always-on error message or is the error message only shown when our jQuery validate library fires and flags up an error?

If it is always on, probably better to use a `label` to add help text to the field. If it is triggered on save, then yeah go for it!

Reply:
There is a server-side validation and a javascript validation, so in theory the server-side one wouldn't ever fail. Server-side is here:
https://github.com/asmecher/sword/blob/master/controllers/grid/form/SwordDepositPointForm.inc.php#L47-L52

Nate, the JS error messages come on and off depending on the state of the form. Each time one of the three fields changes, the form is checked for validity and, if it fails, each empty input gets an error message:
https://github.com/asmecher/sword/blob/master/js/SwordEditDepositPointsFormHandler.js#L47-L64
The Submit button is deactivated while the form is invalid.